tests/phpunit/tests/functions/wpTriggerError.php

```diff
@@ -82,21 +82,31 @@ public function test_should_trigger_deprecation( $function_name, $message, $expe
 	 */
 	public function data_should_trigger_error() {
 		return array(
-			'function name and message are given' => array(
+			'function name and message are given'          => array(
 				'function_name'    => 'some_function',
 				'message'          => 'expected the function name and message',
 				'expected_message' => 'some_function(): expected the function name and message',
 			),
-			'message is given'                    => array(
+			'message is given'                             => array(
 				'function_name'    => '',
 				'message'          => 'expect only the message',
 				'expected_message' => 'expect only the message',
 			),
-			'function name is given'              => array(
+			'function name is given'                       => array(
 				'function_name'    => 'some_function',
 				'message'          => '',
 				'expected_message' => 'some_function(): ',
 			),
+			'allowed HTML elements are present in message' => array(
+				'function_name'    => 'some_function',
+				'message'          => '<strong>expected</strong> the function name and message',
+				'expected_message' => 'some_function(): <strong>expected</strong> the function name and message',
+			),
+			'disallowed HTML elements are present in message' => array(
+				'function_name'    => 'some_function',
+				'message'          => '<script>alert("expected the function name and message")</script>',
+				'expected_message' => 'some_function(): alert("expected the function name and message")',
+			),
```

Comment:
Please add a test case with a link in it to check whether it survives the Kses sanitization mentioned in https://github.com/WordPress/wordpress-develop/pull/6671/files#r1628482559. Like try adding a test with a link to see if it fails then apply the above suggestion if it does.